app/Helpers/render/comment.php

```diff
@@ -163,7 +164,7 @@ function RenderCommentInputRow(string $user, int $articleTypeId, int $articleId)
 {
     sanitize_outputs($user, $formStr);
     $commentId = "art_{$articleTypeId}_{$articleId}";
-    $submitImageUrl = asset('assets/images/icon/submit.png');
+    $submitImage = Blade::render('<x-fas-square-plus class="w-5 h-5" alt="Submit"/>');
```

Comment:
Similarly, Lighthouse is unhappy here:

![Screenshot 2024-01-08 at 5 37 53 PM](https://github.com/RetroAchievements/RAWeb/assets/3984985/ac97bf6a-6a0e-424a-a74e-ea5de187a70e)

I think for this to pass the a11y audit, we'll need to remove the `alt` attribute from the icon and instead use `aria-label` and `title` on the `<button />` tag (both properties should be on the button tag and have the same value).

Note, I believe this a11y check has been failing this entire time. I'm only raising this because IMO it's critical to clean up while we're giving these pieces of the UX some love.